www/pages/guides/now.md

```diff
@@ -0,0 +1,39 @@
+---
+title: 'Deploy on Now'
+menu: side
+linkheadings: 3
+index: 5
```

Comment:
_docs/index.md_ already has an [`index` of 5](https://github.com/ProjectEvergreen/greenwood/blob/master/www/pages/guides/index.md)?  

@hutchgrant 
It seems everything is still fine though in the [preview environment](https://github.com/ProjectEvergreen/greenwood/blob/master/www/pages/guides/index.md)?
<img width="1655" alt="Screen Shot 2020-05-12 at 9 31 11 PM" src="https://user-images.githubusercontent.com/895923/81761475-03ec9e80-9498-11ea-9d78-6ad8193319da.png">

Similar thing [in the Netlify PR](https://github.com/ProjectEvergreen/greenwood/pull/343/files#diff-39ae8f2d113ec3c51153652a81893f08).

I think this should be 6 and the other one should be 7?  Or should _index.md_ even need an index?

Reply:
Hmm, the ordering is inconsistent in other sections as well.

1. About pages are `0` based
1. Getting Started has two `1` indexed files.  Also why would index by `3`?

Maybe a quick PR is needed to set the orders?  

🤔

Reply:
Oooo, oops.  Some of those are for the navigation menu.  Sorry, sorry, 🤦